definitions/tools/vep.cwl

```diff
@@ -14,7 +14,7 @@ requirements:
       ramMin: 64000
       tmpdirMin: 25000
     - class: DockerRequirement
-      dockerPull: "mgibio/vep_helper-cwl:1.1.0"
+      dockerPull: "mgibio/vep_helper-cwl:vep_101.0_v1"
```

Comment:
Do we have a cache dir ready for version 101?

Reply:
yes, it exists, though it hasn't been synced to core yet. We may end up bumping again to 102, as we think it's going to solve an issue with pvactools, but for now, we'll push out 101